ogc-custom-test/src/test/java/org/orbisgis/ogccustomtest/OGC_TestFilterEncoding.java

```diff
@@ -0,0 +1,179 @@
+package org.orbisgis.ogccustomtest;
+
+import net.opengis.fes._2_0_2.*;
+import org.junit.*;
+
+import javax.xml.bind.*;
+import java.io.*;
+
+/**
+ * Created by quillien on 05/05/2017.
```

Comment:
For the author name, in all the OrbisGIS classes the class header has this pattern : 
```java
/**
 * The class description
 *
 * @author your name
 */